node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -262,26 +225,24 @@ pub fn server_initializer_collected_params<'a>(
     let environment_vcl = EnvironmentVcl::new(&app);
     let commandline_vcl = CommandLineVcl::new(args.to_vec());
     let (unspecified_vec, specified_vec) = extract_values_vcl_fill_multiconfig_vec(
```

Comment:
Here's the place where I meant to suggest that you make a `MultiConfig` before making the call.